docs/gui/job-details.rst

```diff
@@ -1,24 +1,18 @@
-gui/workshop-job
-================
+gui/job-details
+===============
 
 .. dfhack-tool::
-    :summary: Adjust the input materials used for a job at a workshop.
-    :tags: unavailable fort inspection jobs
+    :summary: Adjust the input materials and traits used for a job or manager order.
+    :tags: fort inspection jobs workorders interface
 
-This tool allows you to inspect or change the input reagents for the selected
-workshop job (in :kbd:`q` mode).
-
-.. image:: /docs/images/workshop-job.png
+This tool allows you to inspect or change the input reagents for the selected job:
+open the job's "Details" screen by clicking the magnifying glass, then press :kbd:`Ctrl-D`.
```

Comment:
With new focus strings it doesn't work from global jobs list anymore, I'll add it back.
I also want to add it to work order configuration view, because it can be opened for every single order, while the details view only for some.